README.md

```diff
@@ -21,19 +21,18 @@ The following steps will help you to setup Sunbird RC and Esignet services using
     ```bash
     cd inji-certify/docker-compose/docker-compose-certify
     ```
-2. Change the variable `active_profile_env` in [esignet](docker-compose/docker-compose-certify/docker-compose.yml#L79) and [certify](docker-compose/docker-compose-certify/docker-compose.yml#L103) to `active_profile_env=default,mock-identity`
```

Comment:
Can we have a requriements, build, run sections also in the readme file ? Without this it looks like it readme is very specific to docker compose.